backend/kafka/kafka_external_consumer.py

```
@@ -0,0 +1,147 @@
+                data["metric_name"],
+                data["value"],
+                data["timestamp"],
+            )
+            conn.commit()
+
+        except Exception as e:
+            print(f"❌ DB Insert Error: {e} | Data: {data}")
+            try:
+                conn.rollback()
+            except Exception:
+                pass
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)